src/platform/plugins/shared/controls/public/controls/data_controls/options_list_control/components/options_list_popover_invalid_selections.tsx

```diff
@@ -55,7 +55,7 @@ export const OptionsListPopoverInvalidSelections = () => {
     const options: EuiSelectableOption[] = Array.from(invalidSelections).map((key) => {
       return {
         key: String(key),
-        label: fieldFormatter(key),
+        label: fieldFormatter?.(key) ?? String(key),
```

Comment:
nit: instead of having the fieldFormatter be potentially undefined, it might be better to fall back to a `noop` function so you don't have so many nullish coalescing operators.